dipper.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "竹林里有冰 (zhullyb)",
+        "oem": "Xiaomi",
+        "device": "Xiaomi MI 8",
+        "filename": "crDroidAndroid-12.0-20220206-dipper-v8.1.zip",
+        "download": "https://sourceforge.net/projects/zhullyb/files/dipper/12.0/crDroid/crDroidAndroid-12.0-20220206-dipper-v8.1.zip/download",
+        "timestamp": 1644079837,
+        "md5": "83a04b14f1791e4cd457dfc173ce3d4c",
+        "sha256": "8eb407226a2599d06773ab4088bb355eae4e9842b8a26b3432b1b23def189161",
+        "size": 978694978,
+        "version": "8.1",
+        "buildtype": "Monthly",
+        "forum": "https://forum.xda-developers.com/t/rom-official-12-0-crdroid-dipper.4398843/",
+        "gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-S/",
+        "firmware": "https://xiaomifirmwareupdater.com/firmware/dipper/",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://dl.twrp.me/dipper/",
+        "paypal": "",
+        "telegram": "https://t.me/zhullyb_release",
+        "dt": "https://github.com/Bamice/android_device_xiaomi_dipper",
```

Comment:
save dt to crDroid github as per rules
https://github.com/crdroidandroid/rules-and-guidelines#general-rules